src/help/components/contextual/NavigationBar.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import { History, MemoryHistory } from 'history';
+import { Button, ExternalLinkIcon } from 'franklin-sites';
+import cn from 'classnames';
+
+import { LocationToPath, Location } from '../../../app/config/urls';
+
+import helper from '../../../shared/styles/helper.module.scss';
+import styles from './styles/navigation-bar.module.scss';
+
+type Props = {
+  localHistory: MemoryHistory;
+  globalHistory: History;
+};
+
+const NavigationBar = ({ localHistory, globalHistory }: Props) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, setLocalLocation] = useState(localHistory.location);
```

Comment:
Why do we need this?

Reply:
(please add comment)

Reply:
Added a lengthy comment explaining why this is needed